namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/kubernetes/resources.scala

```diff
@@ -1,9 +1,9 @@
 package io.buoyant.namerd.storage.kubernetes
 
 import com.fasterxml.jackson.core.`type`.TypeReference
-import io.buoyant.k8s.{Client, NsThirdPartyVersion, ThirdPartyVersion}
+import io.buoyant.k8s.{Client, NsCustomResourceVersion, CustomResourceVersion}
```

Comment:
Once again, since not all users will be immediately upgrading to a version of Kubernetes that supports CRDs, we need to support both TPRs and CRDs. Thus we can't just replace the third party resource code, we need _both_.

Reply:
Given the only differences between the TPRs and CRDs in `k8s/src/main/scala/io/buoyant/k8s/resources.scala` is the name of the classes/objects, I'd argue that the upgraded version is backwards compatible. (lmk if I'm missing something)

Reply:
@esbie yeah, @deebo91 explained to me that the APIs are equivalent, so I retract that comment. I do think it ought to be documented in that these are equivalent, though.